ga4gh/datarepo.py

```diff
@@ -23,6 +24,8 @@ def __init__(self):
         self._referenceSetIdMap = {}
         self._referenceSetNameMap = {}
         self._referenceSetIds = []
+        self._ontologyNameMap = {}
```

Comment:
This makes perfect sense to me, it's in direct analogy to `referenceSetNameMap` and `datasetNameMap`: Ontologies are pulled up by their name, so there isn't a case (yet) for an `ontologyIdMap`.